.azure-pipelines/Linux-CI.yml

```diff
@@ -127,8 +127,7 @@ jobs:
 
       # Mypy only works with our generated _pb.py files when we install in develop mode, so let's do that
       pip uninstall -y onnx
-      pip install --no-use-pep517 -e .[mypy]
-      python setup.py --quiet typecheck
+      pip install mypy
```

Comment:
what is the intention behind this change?
pip install --no-use-pep517 -e .[mypy] --> will install onnx from current directory along with dependencies listed in the requirement group [] .i.e mypy in this case

whereas pip install mypy will install mypy

Reply:
Hi @askhade thanks for following up. sorry, I thought I closed out this pr.

The change here is a wip, just to test to see if the latest `mypy` would work with Linux builds. 
It's not the change I want to commit/merge. I will close out this PR and either clean this up or open up a new PR based on work https://github.com/onnx/onnx/issues/3110

Thanks!